ddtrace/contrib/asyncio/patch.py

```diff
@@ -15,13 +16,17 @@ def patch():
     setattr(asyncio, '_datadog_patch', True)
 
     loop = asyncio.get_event_loop()
```

Comment:
*aside:* Should we just be patching `asyncio.base_events.BaseEventLoop.create_task` instead?

Reply:
Not suggesting this as a change here, but an overall change to `asyncio`.

We did have an issue where someone reported a problem since we call `asyncio.get_event_loop()` before they have a chance to.

Reply:
Good point - something to consider as we address the open issue